extensions/eclipse-che-theia-plugin-ext/src/node/che-product-service.ts

```diff
@@ -0,0 +1,74 @@
+/*********************************************************************
+ * Copyright (c) 2019 Red Hat, Inc.
+ *
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ **********************************************************************/
+
+import { injectable } from 'inversify';
+import { CheProductService, ProductInfo } from '../common/che-protocol';
+import * as path from 'path';
+import * as fs from 'fs-extra';
+
+const PRODUCT_JSON = '../../src/resource/product.json';
```

Comment:
do we need src there ?

Reply:
resource directory is in src. Which place is better?